README.md

```diff
@@ -156,6 +157,9 @@ In Windows, the script is generally installed in
 
     C:\>\Python34\python \Python34\Scripts\mypy PROGRAM
 
+If you are on Windows using Python 3.3 or 3.4, and would like to use XML reports
```

Comment:
Isn't this true for all Python versions? On my mac in a venv without lxml I get:
```
$ mypy --cobertura-xml-report mypy-report mypy/
You must install the lxml package before you can run mypy with `--cobertura-xml-report`.
You can do this with `python3 -m pip install lxml`.
```